features/users/controller.js

```diff
@@ -0,0 +1,152 @@
+"use strict";
+var debugEnabled = require("../../config").debug;
+var log = require("color-logs")(debugEnabled, debugEnabled, __filename);
```

Comment:
`color-logs` is essentially the console.log.  If this is a production app, how would you make sure your log data can persist to a log file?

Reply:
I don't have any favorite package for this... I would look for some package that would save the console messages in some log manager such as kibana, but I don't have a ready answer for this question, once I always implemented local solutions, depending on the company/project I was working on.